lib/TeamcityReporter.js

```diff
@@ -33,10 +33,19 @@ class TeamcityReporter {
     item(err, args) {
         if (!this.currItem.passed) {
             const msg = this.escape(this.currItem.failedAssertions.join(", "));
-            const responseCode = (this.currItem.response && this.currItem.response.responseTime) || "-";
+            const responseCode = this.currItem.response.code;
```

Comment:
Since `this.currItem.response` is potentially `undefined` the full check is still required.

Reply:
Good comments, I will update it to 
const responseCode = (this.currItem.response && this.currItem.response.code) || "-";